src/server/endpoints/file/list.py

```diff
@@ -56,11 +56,14 @@ def action(project: Project, props: Optional[Props] = None) -> Result:
 
     # List files
     items: List[models.File] = []
+    is_gitignore = helpers.create_file_filter(project)
     for root, folders, files in os.walk(folder):
         root = Path(root)
         for file in files:
             if file.startswith("."):
```

Comment:
We need to stop ignoring files and folders starting from `.` as we now support gitignore :+1: